sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_v6/schema.yaml

```diff
@@ -2000,37 +2000,37 @@ fields:
   type: BOOLEAN
 - mode: NULLABLE
   name: bookmark_migrations_quantity_chrome
-  type: INT64
+  type: INTEGER
```

Comment:
We use `INT64` everywhere else, what do you think about sticking to that?

Reply:
It looks like we use `INTEGER` in schema definitions:
```
(venv) ➜  bigquery-etl git:(clients_daily_schema_updates) ✗ grep -nr --include schema.yaml INTEGER sql/ | wc -l
    1832
(venv) ➜  bigquery-etl git:(clients_daily_schema_updates) ✗ grep -nr --include schema.yaml INT64 sql/ | wc -l
       0
```
I was surprised but I'd stick to whatever comes out of the generator.

Reply:
Huh me too. Carry on!